### PR TITLE
feat(embodiment): add ABot-M0 support

### PR DESCRIPTION
**Pull request by liuhaoyunBUPT**
### Description

This PR adds ABot-M0 support to RLinf. ABot-M0 is a general-purpose VLA model for robotic manipulation (https://github.com/amap-cvlab/ABot-Manipulation), featuring cross-embodiment training on a unified large-scale dataset, Action Manifold Learning (AML) for efficient and stable continuous action prediction, and a modular perception design that can incorporate VLM semantics and optional 3D priors.

### Main changes

- Register ABot-M0 as a supported embodied model.
- Add ABot-M0 model implementation and RL action head integration.
- Add ABot-M0 training configs.
- Add installation/dependency and Docker wiring for ABot-M0.

### How has this been tested?

This change was validated with an end-to-end smoke training using Ray head mode.  
```
# install
cd /mnt/workspace/liuhaoyun.lhy/code/RLinf
export ABOT_PATH=/mnt/workspace/liuhaoyun.lhy/code/ABot-Manipulation
export VGGT_PATH=/mnt/workspace/liuhaoyun.lhy/code/vggt
bash install.sh embodied --venv .venv --model abot_m0 --env maniskill_libero --install-rlinf
source .venv/bin/activate
python -c "import rlinf; import ABot; import vggt; print('IMPORT_SMOKE_OK')"

# smoke test
export REPO_PATH=$(pwd)
export EMBODIED_PATH=$(pwd)/examples/embodiment
export PYTHONPATH=${REPO_PATH}:$PYTHONPATH
export CUDA_VISIBLE_DEVICES=0,1
export MUJOCO_GL=egl
export PYOPENGL_PLATFORM=egl
export ROBOT_PLATFORM=LIBERO
ray stop || true
ray start --head --port=6379

python examples/embodiment/train_embodied_agent.py \
  --config-name libero_10_ppo_abot_m0_2gpu_smoke \
  runner.max_epochs=3 \
  algorithm.rollout_epoch=2 \
  env.train.total_num_envs=4 \
  env.eval.total_num_envs=2 \
  actor.micro_batch_size=2 \
  actor.global_batch_size=8

ray stop
```
### Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Documentation update (Document-only update)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

### Checklist:

- [x] My code follows the code style of this project.
- [x] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

**Comment by zoeyuchao** [comment]
@liuhaoyunBUPT Thanks for your contribution！Copilot adds some comments here, could you solve these comments?

**Comment by zoeyuchao** [comment]
@liuhaoyunBUPT Also, could you provide a doc for this pr? like: https://rlinf.readthedocs.io/en/latest/rst_source/examples/embodied/lingbotvla.html. According this doc, we could reproduce your installation and results. Thanks!

File: rlinf/models/embodiment/abot_m0/abot_m0_action_model.py [hunk 1]
```diff
@@ -0,0 +1,474 @@
...
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""ABot-M0 policy wrapper for RLinf embodied RL.
+
+Provides rollout action generation and training-time logprob recomputation
+through the `BasePolicy` interface.
+"""
+
+from pathlib import Path
```
[the remark this answers is not in the document]
Comment (liuhaoyunBUPT) [reply rlinf/models/embodiment/abot_m0/abot_m0_action_model.py]: Unused import deleted.

File: rlinf/models/embodiment/abot_m0/action_head_rl.py [hunk 2]
```diff
@@ -0,0 +1,282 @@
...
+# limitations under the License.
+
+"""RL action head wrapper for ABot-M0."""
+
+import random
+from typing import Any, Literal, Optional
+
+import torch
+import torch.nn as nn
+from torch.distributions import Normal
```
[the remark this answers is not in the document]
Comment (liuhaoyunBUPT) [reply rlinf/models/embodiment/abot_m0/action_head_rl.py]: Unused import deleted.

File: examples/embodiment/config/model/abot_m0.yaml [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+add_value_head: true
+
+# RL head configuration
+rl_head_config:
+  joint_logprob: false
+  noise_method: "flow_sde"
+  noise_level: 0.5
+  ignore_last: true
+  add_value_head: ${actor.model.add_value_head}
+  t_eps: 5.0e-2
```
[the remark this answers is not in the document]
Comment (liuhaoyunBUPT) [reply examples/embodiment/config/model/abot_m0.yaml:29]: Thanks for catching this. I wired rl_head_config.t_eps into the ABot-M0 RL head and propagate it to the base action head during initialization, so the config value is now actually used. I kept the config field and removed the misleading behavior.

File: rlinf/workers/rollout/hf/huggingface_worker.py [hunk 4]
```diff
@@ -341,7 +342,7 @@ async def sync_model_from_actor(self):
             async_op=True,
             options=self._sync_weight_comm_options,
         ).async_wait()
-        self.hf_model.load_state_dict(param_state_dict)
+        self.hf_model.load_state_dict(param_state_dict, strict=False)
```
[the remark this answers is not in the document]
Comment (liuhaoyunBUPT) [reply rlinf/workers/rollout/hf/huggingface_worker.py]: I have removed strict=False from actor→rollout weight sync, so rollout now uses strict loading again and will fail fast on real mismatches.
The earlier ABot-M0 actor/rollout key mismatch is now handled in the model-side load_state_dict normalization/alias logic (strip_known_prefixes + get_alias_key), with explicit validation that raises on any non-allowed missing/unexpected keys.
So we keep safety at sync time and only allow the intended ABot-M0 compatibility mapping.

**Comment by liuhaoyunBUPT** [comment]
@zoeyuchao Thanks for the review! I have addressed the Copilot comments and updated the PR accordingly.

I also added the ABot-M0 documentation, including setup and smoke test instructions, and updated the related English/Chinese doc index pages.

Please take another look. Thanks!

**Comment by zoeyuchao** [comment]
@liuhaoyunBUPT Thanks again! The ci shows there are still some errors in this pr, such as lint check and commit check. could you fix them?
